### DISCUSSION TITLE
Mountain Duck is unable to connect WebDAV of Pikpak

### DISCUSSION THREAD

**Discussion by DannyDeng2023** [iterate-ch/cyberduck#16167] [Q&A]
I'm trying to connect Pikpak Webdav by using Mountain Duck on Mac mini (Sonoma 14.5)    it shows error "Unexpected response (400 Bad Request)" . However it's OK when I use infuse to connect Pikpak Webdav by the same way.

**Comment by tatze96** [comment]
Please [enable debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the mentioned error and send us the log files with the ticket reference GH-16167 to support@mountainduck.io for further investigation.

Ensure to add a screenshot of the error message to the email.

**Reply by sH1222J** [reply]
This issue was reported more than a year ago, but it seems it still hasn’t been resolved. Is there any update on this?

**Comment by dkocher** [comment]
The issue is that `dav.mypikpak.com` responds with a `400 Bad Request` response to a `HEAD` request. Please report this bug to the developers of Pikpak.
